File: Programming Language(ver.2)/C++/301.cpp
#include <iostream>
using namespace std;

int compute(int []);
int main()
{
    int input[2];
    cin >> input[0] >> input[1];
    cout << compute(input);
    return (0);
}

int compute(int s[])
{
    int i, j;
    for (i = 1; i <= s[1]; i++) {
        for (j = 1; j <= s[0]; j++) {
            cout << "*";
        }
        cout << endl;
    }
    return s[0] * s[1];
}

File: Programming Language(ver.2)/Python/301.py
def compute(arr):
    W = arr[0]
    H = arr[1]
    total_stars = 0
    for i in range(H):
        for j in range(W):
            print('*', end='')
            total_stars += 1
        print()
    return total_stars

input_array = []
input_array.append(int(input()))
input_array.append(int(input()))
total_stars = compute(input_array)
print(total_stars)